.env.example

```diff
@@ -2,17 +2,19 @@
 # React development variables
 REACT_APP_MARKET_API_URL=https://api.coingecko.com/api/v3
 
-REACT_APP_SUBGRAPH_URL_1=
-REACT_APP_SUBGRAPH_URL_42=
-REACT_APP_SUBGRAPH_URL_LOCAL={local-subgraph}
+REACT_APP_SUBGRAPH_URL_1="https://api.thegraph.com/subgraphs/name/balancer-labs/balancer"
```

Comment:
Are these supposed to have balancer subgraph addresses?

Reply:
No idea, I dont knwo for what we used this env vars.